documentation/modules/mtv-shared-disks.adoc

```diff
@@ -0,0 +1,224 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtv-shared-disks_{context}"]
+= Migrating virtual machines with shared disks
+
+You can migrate VMware virtual machines (VMs) with shared disks using the {project-first}. This functionality is available for cold migrations only. 
+
+Shared disks are disks that are attached to multiple VMs and that use the multi-writer option. As a result of these characteristics, shared disks are difficult to migrate. 
+ 
+In certain situations, applications in VMs require shared disks. Databases and clustered file-systems are the primary use cases for shared disks.
+
+{project-short} version 2.7.11 or later includes a parameter named `migrateSharedDisks` in `Plan` CRs that instructs {project-short} to either migrate shared disks or to skip them during migration as follows:
+
+* If set to `true`, {project-short} uses the regular cold migration flow using `virt-v2v` and labeling the shared PVC. This is the default setting. 
+
+* If set to `false`, {project-short} uses the KubeVirt CDI for disk transfer, and skips the shared disks. After the disk transfer, {project-short} automatically attempts to locate the already shared PVCs and the already migrated shared disks and attach them to the VMs. 
+
+By default, `migrateSharedDisks` is set to `true`.
+
+To successfully migrate VMs with shared disks, create two `Plan` CRs as follows: 
+
+* In the first, set `migrateSharedDisks` to `true`. {project-short} migrates the following:
+
+** All shared disks.
+** For each shared disk, one of the VMs that is attached to it. If possible, choose VMs so that that plan does not contain any shared disks that are connected to more than one VM in the plan. See the figures below for further guidance.
+** All unshared disks attached to the VMs you choose for this plan. 
+
+* In the second, set `migrateSharedDisks` to 'false`. {project-short} migrates the following:
+
+** All other VMs
+** The unshared disks of the VMs in the second `Plan` CR
+
+To illustrate the division of VMs and disks between `Plan` CRs, consider the two figures that follow: 
+
+In the first figure, the first `Plan` CR migrates VMs 1 and 3, shared disks 1, 2, and 3, and the non-shared disks of VMs 1 and 3. VMs 1 and 3 were chosen because together they connect to all the shared disks. In practice, {project-short} first migrates VM 1 and all disks attached to it, then does the same with VM 3.Choosing VM 1 and VM 3 and only those two ensures that all the shared disks are migrated, but only once.  
```

Comment:
This is missing the reference to the figure

Reply:
This figure will be replaced to fit documentation requirements for figures.